src/Models/ShallowWaterModels/shallow_water_advection_operators.jl

```diff
@@ -77,13 +95,19 @@ a velocity field U = (u, v), ∇·(Uc),
 
 which will end up at the location `ccc`.
 """
-@inline function div_Uc(i, j, k, grid, advection, solution, c)
-    1/Vᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, transport_tracer_flux_x, advection, solution.uh, solution.h, c) +        
-                             δyᵃᶜᵃ(i, j, k, grid, transport_tracer_flux_y, advection, solution.vh, solution.h, c))
+
+@inline function div_Uc(i, j, k, grid, advection, solution, c, formulation)
+    1/Azᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, transport_tracer_flux_x, advection, solution[1], solution.h, c) +        
+                              δyᵃᶜᵃ(i, j, k, grid, transport_tracer_flux_y, advection, solution[2], solution.h, c))
```

Comment:
Shouldn't we multiply by length, ie

```suggestion
    1/Azᶜᶜᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Δy_qᶠᶜᶜ, transport_tracer_flux_x, advection, solution[1], solution.h, c) +        
                              δyᵃᶜᵃ(i, j, k, grid, Δx_qᶜᶠᶜ, transport_tracer_flux_y, advection, solution[2], solution.h, c))
```

?

Reply:
Same for below